src/sherpa_ai/database/user_usage_tracker.py

```diff
@@ -111,9 +147,17 @@ def get_data_since_last_reset(self, user_id):
                 "token": item.token,
                 "timestamp": item.timestamp,
                 "reset_timestamp": item.reset_timestamp,
+                "reminded_timestamp": item.reminded_timestamp,
             }
             for item in data
         ]
+    
+    def check_if_reminded(self,combined_id):
```

Comment:
We still have the problem here that Slack User IDs are allowed to contain an underscore `_`. For this reason it isn't safe to combine the user ID and team ID with an underscore. We should use a different separator or treat the user ID and team ID as distinct pieces of data instead of combining them.

This affects several APIs.

Reply:
I checked and this is not a Slack user ID. This is what the Slack system generates for that specific user and its all cap no special character ID we are safe to use underscore or any special character as a separator. @oshoma

Reply:
Ah, I was mixing up Slack User Names and User IDs. Best answer I could find was [here](https://stackoverflow.com/questions/51207581/can-i-differentiate-a-slack-display-name-from-a-slack-id) which confirms your finding. User names can contain underscores, and User IDs do not. Thanks for checking!